indico/modules/events/serialize.py

```diff
@@ -0,0 +1,76 @@
+from icalendar import Calendar, Event
+from lxml import html
+from lxml.etree import ParserError
+from werkzeug.urls import url_parse
+
+from indico.core import signals
+from indico.core.config import config
+from indico.util.date_time import now_utc
+from indico.util.signals import values_from_signal
+
+
+def event_to_ical(event, user=None):
+    """Serialize an event into an iCalendar.
```

Comment:
hm... "into an iCalendar" vs "to ical"? No strong opinion. I wonder if people commonly use the "iCalendar" term when not talking about the actual iCalendar application on macos

Reply:
Yup, and iCalendar is the specification, files are `ical` extension.